zhaquirks/xiaomi/__init__.py

```diff
@@ -409,6 +409,22 @@ def pressure_reported(self, value):
         self._update_attribute(self.ATTR_ID, value)
 
 
+class AnalogInputCluster(CustomCluster, AnalogInput):
```

Comment:
This one should be renamed somehow, WDYS?

Reply:
BTW, does it make sense to reuse it in the relay quirk? @dmulcahey

Reply:
Why do you want to rename this?

Reply:
If you want to move this up to `__init__` and share it that's fine

Reply:
Because it's name is too generic and might be confusing

Reply:
I don't understand... it's a custom implementation of the `AnalogInput` cluster... hard to get more specific. If you mean that it doesn't convey that it relays power to EM cluster then add a comment at the top and that is sufficient. The goal is for people to know what ZCL clusters they have when looking at the replacement dict so the names should stay as close to the ZCL cluster names as possible.

Reply:
Got it, thanks!